src/indexOf.js

```diff
@@ -6,7 +6,8 @@ var _isArray = require('./internal/_isArray');
 /**
  * Returns the position of the first occurrence of an item in an array,
  * or -1 if the item is not included in the array. `R.equals` is used to
- * determine equality.
+ * determine equality in most cases. When target is a function `R.identical`
+ * is used to determine if the target is equal to an element in the array.
```

Comment:
This isn't quite right. `R.equals` is used to determine equality in all cases, but `R.equals` itself compares functions by identity. For example:

```javascript
var f = function() {};

R.equals(f, f);  // => true
R.equals(function() {}, function() {});  // => false
```

Perhaps we should clarify the description of `R.equals` instead.